Stack/Opc.Ua.Core/Stack/Types/ContentFilter.Evaluate.cs

```diff
@@ -417,6 +423,17 @@ private static bool IsEqual(object value1, object value2)
                 return false;
             }
 
+            //check for strings
```

Comment:
the comparison above with DBNull can be removed. It must have slipt in on the first port to .NET Core.